adb/adb_protocol.py

```diff
@@ -277,9 +277,10 @@ def Connect(cls, usb, banner=b'notadb', rsa_keys=None, auth_timeout_ms=100):
       InvalidResponseError: When the device does authentication in an
           unexpected way.
     """
+    hostbanner = 'host::%s' % banner
```

Comment:
I'm trying to fix remote debug. The URL handler expects a string. The and command expects bytes and the protocol expects bytes. I was playing with different options.